WORKSPACE

```diff
@@ -87,12 +87,14 @@ rust_register_toolchains(
     ],
 )
 
-BAZEL_VERSION = "5.2.0"
+BAZEL_VERSION = "6.5.0"
 
-BAZEL_INSTALLER_VERSION_LINUX_X86_64_SHA = "7d9ef51beab5726c55725fb36675c6fed0518576d3ba51fb4067580ddf7627c4"
+BAZEL_INSTALLER_VERSION_LINUX_X86_64_SHA = "c0161a346b9c0d00e6eb3d3e8f9c4dece32f6292520248c5ab2e3527265601c1"
 
-BAZEL_INSTALLER_VERSION_LINUX_ARM64_SHA = "ae50cb7d64aebee986287134ff8ca0335651a0c1685348b3216f3fdfa20ff7e7"
+# I couldn't find any official checksums for the LINUX_ARM64 version of the Bazel installer in
+# https://github.com/bazelbuild/bazel/releases/tag/6.5.0.
+BAZEL_INSTALLER_VERSION_LINUX_ARM64_SHA = "5afe973cadc036496cac66f1414ca9be36881423f576db363d83afc9084c0c2f"
```

Comment:
I used the [bazel-6.5.0-linux-arm64.sha256](https://github.com/bazelbuild/bazel/releases/download/6.5.0/bazel-6.5.0-linux-arm64.sha256) sha instead but this will most likely not work when trying to install Bazel on a linux-arm64 instance.

I'm not sure what workaround we can use?

Reply:
How is CI passing right now? Don't we test on arm?

Reply:
The log output from buildkite suggests that this is working fine:

https://buildkite.com/sorbet/sorbet/builds/30965#018ecf5d-ea61-4039-9496-9a3b821c3b4e/180-189

So I'm curious if you could elaborate on what about this you think is "not likely to work."

Reply:
As you pointed out on Slack, this checksum is actually not used for `linux-arm64` as there is no binary installer for this platform: https://github.com/sorbet/sorbet/blob/master/bazel#L102.

I wonder if we could just remove the `BAZEL_INSTALLER_VERSION_LINUX_ARM64_SHA` constant and leave a comment explaining this instead as this is confusing.

Reply:
Ok, we actually need `BAZEL_INSTALLER_VERSION_LINUX_ARM64_SHA` as it's used to checksum the installed bazel without the installer: https://github.com/sorbet/sorbet/blob/master/bazel#L73.

I updated the comment 👍